sdk/core/core-http/review/core-http.api.md

```diff
@@ -157,12 +156,6 @@ export const Constants: {
 // @public (undocumented)
 export function createPipelineFromOptions(pipelineOptions: InternalPipelineOptions, authPolicyFactory?: RequestPolicyFactory): ServiceClientOptions;
 
-// @public
-export function createSpanFunction({ packagePrefix, namespace }: SpanConfig): <T extends OperationOptions>(operationName: string, operationOptions: T) => {
```

Comment:
Since this is a major breaking change for `core-http` it may be a good idea to update the package version and changelog now to make sure this breaking change doesn't slip as a minor one

Reply:
If we don't want to take a major change with this. It may be a good idea to add some kind of warning that this would go away in the next major version. Maybe as part of the function TSDOC and maybe logging a warning?

@xirzec any thoughts?

Reply:
I added it back because all your points were good. I'm mostly concerned with our packages consuming it since they call it unconditionally but I'm not sure if we ever represented to anyone external from the SDK team that these were for public consumption.

Then again, perhaps we didn't explicitly signal otherwise.

Reply:
Yeah I think we're stuck with it for now, since we had at least one client package ship a beta that depends on it